README.md

```diff
@@ -57,7 +57,7 @@ was used.
 To use it, add it to your Gemfile:
 
 ```ruby
-gem 'acts-as-taggable-on', '~> 8.0'
+gem 'acts-as-taggable-on', '~> 9.0'
```

Comment:
I think `~> 9.0` should be `~> 8.1`?

Reply:
why ? 
master branch is 9.0.0

Reply:
Yesterday I tried to use `acts_as_taggable_on` and following the README I added `gem 'acts-as-taggable-on', '~> 9.0'` to my Gemfile and ran `bundle`.

I got this error;
`Could not find gem 'acts-as-taggable-on (~> 9.0)'`

Seems rubygems.org doesn't have 9.0.0. The latest it has is 8.1.0;
https://rubygems.org/gems/acts-as-taggable-on